examples/firmware_drivers/data_source.tf

```diff
@@ -1,17 +1,17 @@
 provider "oneview" {
-	ov_username   = var.username
-	ov_password   = var.password
-	ov_endpoint   = var.endpoint
-	ov_sslverify  = var.ssl_enabled
-	ov_apiversion = var.api_version
-	ov_ifmatch    = "*"
-  }
-  
+  ov_username   = var.username
+  ov_password   = var.password
+  ov_endpoint   = var.endpoint
+  ov_sslverify  = var.ssl_enabled
+  ov_apiversion = var.api_version
+  ov_ifmatch    = "*"
+}
+
 # Testing data source
 data "oneview_firmware_drivers" "drivers" {
-  id = "Synergy_Custom_SPP_2021_02_01_Z7550-97110"
+  id = "HPE Synergy Service Pack,SY-2021.11.01"
```

Comment:
have u checked if space is given after ',' . I mean generally ppl use that way...in our implementation...it should strip off those extra spaces after ,

Reply:
yeah done

Reply:
yes it is added